### ISSUE TITLE
Make Loyalty Implants actually do something outside rev/cult

### ISSUE THREAD

**Issue by ParagonPrime** [yogstation13/yogstation-classic#733]
### Expected Behaviour

Loyalty implants instantly make the implantee HEIL NANOTRASEN and abandon former loyalties
### Actual Behaviour

Every tator on station gets sec access from the dumbass HoP, knowing full well that they can take implants **(without resisting them visibly)** and suddenly have free access to stun weapons, armor, the armory, and the best alibi for stunning and dragging people away possible
### Steps to reproduce the problem
1. Implant person-secretly-tator
2. Get tasered and murdered by same person-secretly-tator
3. Be pissed

---

Okay, but in all seriousness, I'm tired of being Warden/HoS and losing my shit at HoPs who promote random assistants to Sec and then force me to deal with them potentially stabbing me in the back at any moment, as they do 99% of the time.

"BUT THAT'S META!!", you cry! No. It isn't- it's common sense. **NanoTrasen wouldn't randomly allow Command to recruit random people on their station to be sec officers**, and especially not without an assurance that their fucking loyalty implants actually _work_

So, two possible fixes:
1. Being implanted **INSTANTANEOUSLY MAKES THE IMPLANTEE LOYAL TO NANOTRASEN**, forfeiting former traitor/ling objectives. Might end up buffing implants too much, though (on the other hand- Security can now get their very own pet Changeling for those plainclothes operations!)
2. Being implanted as a Traitor/Ling **DISPLAYS THE SAME RESIST IMPLANT AS REVHEADS/GANGBOSSES**. Slightly less fun, but slightly less of a buff too.

Either way, I think this is necessary.
That, or I'm stunbattoning and booting out anyone HoP sends my way unless we're **really** in an emergency.

**Comment by ShadowDeath6** [comment]
just have traitors resist shit, removing RNG'd antag status would be retarded

**Comment by ToGWtF** [comment]
Let me channel my inner Badam

Think about it. Traitors are already sleeper agents/undercover/already neutral enough to NT that why would work for the Syndicate that already got passed the corporation. It makes sense they could pass a screening to become what is essentially mallcops on what is supposed to be a research station. In fact, SS13 is supposed to be NT's most under-funded station (Come on, we don't even have more than what, six beds? Four showers? Life there must suuuuck)

Not to mention, Syndicates have their own Mindslave implant that's stronger and forces people to their will. Gangs even have an implant-breaker, and can subvert even the highest authority, the Captain,  to their will. I wouldn't put it past the syndicate to have a way to counter the implants. Whether through psychological conditioning (Sleeper agent theory) , their own pre-installed implants (Mindslave implant), or just by sheer will (They have my daughter, if I don't obey the syndicate I'll never see her again) they can null the effects of the Loyalty implants grasp on their mind.

Changelings aren't even fucking human. They can communicate in non-verbal, instant hivemind. Their brains obviously work on levels much higher than ours. The loyalty implant is keyed into Human brain function (Only once the furries found out about the game did we get other species). Changelings aren't even the person they start the game as. That's just another person they've absorbed and replaced. They have their DNA, their memories, their personalities dialed in perfectly, and yet they keep their Changeling will. They have complete control over all their biological functions and can change their physical form at will. They can beat a microchip poked into the back of their skull.

I'm actually up for giving Traitors a chance to resist the implant, without exactly giving them away (If you can't bullshit your way into them testing a second implant, how did you think your plan was going to work anyway? If it fails a second time, sucks for youuuu). Changelings, no. Just wouldn't even affect them.

Buuuuut

What I do want, is that asshole Graytider you implanted after his third stay in the brig for assaulting security officers shouting "SHITSEC!!" to no longer be able to continue shitting up the place.

Sure, what he's doing isn't exactly breaking any major rules, but any admin can check if he's implanted and they can no longer rule it an "IC issue". They player fucked up enough to get himself stuck with a fucking mind control implant shoved into his spinal column and now has play nice. Give the security players some way of actually handling a terrible player and making his constant aggressive, yet not exactly rule breaking, behavior worthy of an actual report. By the same hand, sec implanting you for no reason would be something to report, and in either way would bring the situation to their attention to keep an eye on the players involved.

TL;DR
Traitors stay traitor
Traitors have a chance to resist and get caught
Changelings laugh at the thought of it
Loyalty implants force non-antags to stop being shit by being hounded by staff

**Comment by LesVegetables** [comment]
This seems more like a problem of incompetence on HOP's part, whenever I'm hop I ask for huds and tell any candidates for mall-cop to get implanted BEFORE giving them access.

This would also just waste implants, every toolbox slinging moron would be implanted and probably suicide right afterwards

**Comment by EMT321** [comment]
I firmly agree: make traitors resist implants, even if theres no dialogue for it. Just that the implant doesn't take. Still lets people talk their way out of it, and requires observation to catch.
